litmus-portal/frontend/src/views/Home/PassedVsFailed/index.tsx

```diff
@@ -58,12 +67,18 @@ const PassedVsFailed: React.FC<PassedVsFailedProps> = ({ passed, failed }) => {
             {failedValue === 0 ? (
               <div />
             ) : (
-              <img
-                src="./icons/Fail.png"
-                alt="Failed Icon"
+              <Avatar
+                style={{
+                  backgroundColor: '#CA2C2C',
```

Comment:
put all colors in theme...if it is in react component use "useTheme" if it is used in styles...use normal theme value you get